### PR TITLE
docs: sbt-revoke

### PR DESCRIPTION
**Pull request by robert-zaremba**
Updates revoke API

### CODE REVIEW

File: contracts/registry/src/registry.rs [hunk 1]
```diff
@@ -335,11 +335,12 @@ impl SBTRegistry for Contract {
         SbtTokensEvent { issuer, tokens }.emit_renew();
     }
 
-    /// Revokes SBT by burning it.
+    /// Revokes SBT.
     /// Must be called by an SBT contract.
-    /// Must emit one of `Revoke` or `Burn` event.
+    /// Must emit `Revoke` event.
+    /// Must emit also emit `Burn` event if the SBT tokens are burned (removed).
```
Comment (sczembor) [inline contracts/registry/src/registry.rs:341]:
```suggestion
    /// Must also emit `Burn` event if the SBT tokens are burned (removed).
```

File: contracts/sbt/src/lib.rs [hunk 2]
```diff
@@ -123,11 +123,12 @@ pub trait SBTRegistry {
     /// Must emit `Renew` event.
     fn sbt_renew(&mut self, tokens: Vec<TokenId>, expires_at: u64);
 
-    /// Revokes SBT by burning the token or updating its expire time.
+    /// Revokes SBT.
     /// Must be called by an SBT contract.
-    /// Must emit one of `Revoke` or `Burn` (if the token is removed) event.
+    /// Must emit `Revoke` event.
+    /// Must emit also emit `Burn` event if the SBT tokens are burned (removed).
```
Comment (sczembor) [inline contracts/sbt/src/lib.rs:129]:
```suggestion
    /// Must also emit `Burn` event if the SBT tokens are burned (removed).
```

File: contracts/registry/src/registry.rs [hunk 3]
```diff
@@ -335,11 +335,12 @@ impl SBTRegistry for Contract {
...
     /// Returns true if a token is a valid, active SBT. Otherwise returns false.
```
Comment (sczembor) [inline contracts/registry/src/registry.rs:342]: How about the return value? We are passing multiple tokens to the method

**Review by sczembor — APPROVED** [review]
left few comments

File: contracts/registry/src/registry.rs (hunk 3 above)
Comment (robert-zaremba) [reply contracts/registry/src/registry.rs:342]: true, let's remove return value

**Pull request merged**